### PR TITLE
X, S, Y, STY definitions

### LINKED ISSUE #46: 💡 Better definition of selectivity
### Description

Brought up in #45

### PR DESCRIPTION
**Pull request by dalito**
This issue makes the definition of selectivity universal. It was linked to photo-catalysis before. Moreover, it adds

* converison
* yield
* space-time yield
* catalyst performance measure
* reactor performance measure
* chemical conversion performance measure

Closes #46

### CODE REVIEW

File: vocabularies/voc4cat/0005006.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0005006>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0005006"^^xsd:token ;
+    skos:broader
+        <https://w3id.org/nfdi4cat/voc4cat_0005001> ,
+        <https://w3id.org/nfdi4cat/voc4cat_0005002> ,
+        <https://w3id.org/nfdi4cat/voc4cat_0005003> ;
+    skos:definition """A physical quantity that describes the amount of product prodced per unit of time and unit of producing entity. The producing entity is for example the volume of a chemical reactor or in catalysis the mass or volume or moles of catalyst.
```
Comment (nmoust) [inline vocabularies/voc4cat/0005006.ttl:12]: - There is a typo in: product **prodced**. 
- There are three """ before starting the definition. And there is none at the end of it. I am not sure if this plays a role though.

File: vocabularies/voc4cat/0005004.ttl [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0005004"^^xsd:token ;
+    skos:broader
+        <https://w3id.org/nfdi4cat/voc4cat_0005001> ,
+        <https://w3id.org/nfdi4cat/voc4cat_0005002> ,
+        <https://w3id.org/nfdi4cat/voc4cat_0005003> ;
+    skos:definition "A dimensionless physical quantity describing the fraction of a reactant that reacts in a chemical conversion. If a reactant is consumed completely its conversion is 1 (or 100 %)."@en ;
+    skos:historyNote "0000-0002-5898-1820 David Linke created the resource"@en ;
+    skos:inScheme <https://w3id.org/nfdi4cat/voc4cat_> ;
+    skos:prefLabel "conversion"@en ;
```
Comment (nmoust) [inline vocabularies/voc4cat/0005004.ttl:15]: @dalito: Do you think that we should also include a "chemical conversion""concept?

File: vocabularies/voc4cat/concept_scheme.ttl [hunk 3]
```diff
@@ -12,12 +12,15 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
         <https://w3id.org/nfdi4cat/voc4cat_0001900> ,
         <https://w3id.org/nfdi4cat/voc4cat_0001901> ;
     dcterms:identifier "voc4cat_"^^xsd:token ;
-    dcterms:modified "2024-02-02"^^xsd:date ;
+    dcterms:modified "2024-02-20"^^xsd:date ;
     dcterms:publisher <https://w3id.org/nfdi4cat/> ;
     owl:versionInfo "automatic" ;
     skos:definition "A vocabulary for all areas of catalysis initiated in NFDI4Cat. Starting with photocatalysis we add more and more terms from other areas of catalysis and related disciplines like chemical enineering or materials science."@en ;
```
Comment (nmoust) [inline vocabularies/voc4cat/concept_scheme.ttl:18]: @dalito : There is typo in: ... like chemical **enineering**...

File: vocabularies/voc4cat/0005006.ttl (hunk 1 above)
Comment (dalito) [reply vocabularies/voc4cat/0005006.ttl:12]: This is fine. The closing triple quotes are on the next line.

File: vocabularies/voc4cat/0005004.ttl (hunk 2 above)
Comment (dalito) [reply vocabularies/voc4cat/0005004.ttl:15]: Yes. Could be a nice example to demonstrate how to add/reference a related term. Tbd in separate PR.

**Comment by dalito** [comment]
Thanks for spotting the typos! In turned out that spell checking in my editor was off for turtle-files.

**Review by nmoust — APPROVED** [review]
@dalito: Thank you for the contribution. I will accept the changes.

**Pull request merged**